includes/database/engine/class-query.php

```diff
@@ -50,68 +54,66 @@ class Query extends Base {
 	/**
 	 * Name of the database table to query.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 * @var   string
 	 */
 	protected $table_name = '';
 
 	/**
 	 * String used to alias the database table in MySQL statement.
 	 *
-	 * Keep this short, but descriptive. I.E. "oi" for order items.
+	 * Keep this short, but descriptive. I.E. "tr" for term relationships.
 	 *
 	 * This is used to avoid collisions with JOINs.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 * @var   string
 	 */
 	protected $table_alias = '';
 
 	/**
-	 * Name of class used to setup the database schema
+	 * Name of class used to setup the database schema.
 	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 * @var   string
 	 */
-	protected $table_schema = '\\EDD\\Database\\Schemas\\Base';
+	protected $table_schema = __NAMESPACE__ . '\\Schema';
```

Comment:
For this and `$item_shape` I'm seeing this error:

> expression is not allowed as field default value

Reply:
Aww. That's too bad. I've reverted these and pushed them up here and in Berlin.

I wonder if these errors are related to specific PHP versions. I'm not seeing them anywhere on `7.4.10`.

Reply:
Not sure. PhpStorm was warning me about it. Usually if it's version-specific it says so, but this time it didn't say anything. :woman_shrugging: